cypress/integration/createBoard.spec.js

```diff
@@ -0,0 +1,33 @@
+///<reference types="Cypress" />
+
+import {loginPage} from './../page_objects/loginPage';
+import {createOrganization} from './../page_objects/createOrganization';
+import {createBoard} from './../page_objects/createBoard';
```

Comment:
be more consistant with naming. If you used camel case in 90% of repo, why do we have page_objects